index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Dancing+Script&family=Roboto:wght@700&family=Zen+Dots&display=swap" rel="stylesheet">
+</head>
+<body>
+    <main class="wrapper">
+    <header>
+        <h1>My Logo</h1>
+        <img src="Union.png" alt="navigation icon">
+    </header>
+
+    <section class="section-one">
+        <h1>I'm Salma Glad to see you!</h1>
+        <p>I'm a software developer! 
+            I can help you build a product , 
+            feature or website Look through 
+            some of my work and experience! 
+            If you like what you see and have
+             a project you need coded, 
+            don't hestiate to contact me.</p>
+            <h3>LET'S CONNECT</h3>
+            <img src="ic-social media icons.png" alt="social-media icons">
```

Comment:
- Can we kindly add the rest of the social icons to our portfolio? Do ensure to remember that they should be in a list tag. By this i mean in a `<ul>` and `<li>` tags. 

Also, each icon in an `<li>` should be wrapped inside an `<a>` tag as this `a` tag will let you connect your social networks with the icons, using href. 🥂